src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodsService.scala

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import akka.actor.{Actor, _}
+import com.wordnik.swagger.annotations.{Api, _}
+import org.broadinstitute.dsde.firecloud.MethodsClient
+import org.broadinstitute.dsde.firecloud.model.MethodEntity
+import org.slf4j.LoggerFactory
+import spray.routing.{HttpService, Route}
+
+class MethodsServiceActor extends Actor with MethodsService {
+  def actorRefFactory = context
+  def receive = runRoute(listRoute)
+}
+
+@Api(value = "/methods", description = "Methods Service", produces = "application/json")
```

Comment:
This may be a red herring, but I couldn't get the swagger-ui to work until I added consumes="application/json" as well. Before adding that, I would get a 406 error that mentioned it was expecting text/plain.

HOWEVER, I then removed consumes="application/json" to verify that was the fix ... and it still worked. So I dunno what happened. If it works for you, feel free to ignore.